[2021-12-02 23:40:02,024] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-01T09:00:00+00:00 [queued]>
[2021-12-02 23:40:02,031] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-01T09:00:00+00:00 [queued]>
[2021-12-02 23:40:02,031] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-12-02 23:40:02,031] {taskinstance.py:881} INFO - Starting attempt 7 of 7
[2021-12-02 23:40:02,031] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-12-02 23:40:02,039] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2021-12-01T09:00:00+00:00
[2021-12-02 23:40:02,040] {standard_task_runner.py:54} INFO - Started process 15763 to run task
[2021-12-02 23:40:02,087] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2021-12-01T09:00:00+00:00', '--job_id', '76', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmp4mh5seuy']
[2021-12-02 23:40:02,087] {standard_task_runner.py:78} INFO - Job 76: Subtask twitter_aluraonline
[2021-12-02 23:40:02,105] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-01T09:00:00+00:00 [running]> on host kauai-VirtualBox
[2021-12-02 23:40:02,140] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2021-12-02 23:40:02,142] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-01T09:00:00.00Z&end_time=2021-12-02T09:00:00.00Z
[2021-12-02 23:40:02,388] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-02 23:40:02,769] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-01T09:00:00.00Z&end_time=2021-12-02T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy837untnqw732a5ss50dfvplo59
[2021-12-02 23:40:02,769] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-02 23:40:03,168] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-01T09:00:00.00Z&end_time=2021-12-02T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy835rfddk3s3r4rwa6estz0p9bx
[2021-12-02 23:40:03,169] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-02 23:40:03,557] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-01T09:00:00.00Z&end_time=2021-12-02T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy833nli8zirlepdqcomook1lf25
[2021-12-02 23:40:03,557] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-02 23:40:03,968] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-01T09:00:00.00Z&end_time=2021-12-02T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy833n0b1mhz36iukjkp6fxp18xp
[2021-12-02 23:40:03,968] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-02 23:40:04,363] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-01T09:00:00.00Z&end_time=2021-12-02T09:00:00.00Z&next_token=b26v89c19zqg8o3fpdy833m3w9qcy0pq139bgdjk76dfh
[2021-12-02 23:40:04,364] {logging_mixin.py:112} WARNING - /home/kauai/datapipeline/venv/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-02 23:40:04,705] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20211201T090000, start_date=20211203T024002, end_date=20211203T024004
[2021-12-02 23:40:07,014] {local_task_job.py:102} INFO - Task exited with return code 0
